lib/sisimai/bite/email/ezweb.rb

```diff
@@ -0,0 +1,260 @@
+
+      Re0 = {
+        :'from'       => %r/[<]?(?>postmaster[@]ezweb[.]ne[.]jp)[>]?/i,
+        :'subject'    => %r/\AMail System Error - Returned Mail\z/,
+        :'received'   => %r/\Afrom[ ](?:.+[.])?ezweb[.]ne[.]jp[ ]/,
+        :'message-id' => %r/[@].+[.]ezweb[.]ne[.]jp[>]\z/,
+      }
+      Re1 = {
+        :begin  => %r{\A(?:
+             The[ ]user[(]s[)][ ]
+            |Your[ ]message[ ]
+            |Each[ ]of[ ]the[ ]following
+            |[<][^ ]+[@][^ ]+[>]\z
+            )
+        }x,
+        :rfc822   => %r#\A(?:[-]{50}|Content-Type:[ ]*message/rfc822)#,
+        :boundary => %r/\A__SISIMAI_PSEUDO_BOUNDARY__\z/,
+        :endof    => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
+        # notaccept: [ %r/The following recipients did not receive this message:/ ],
+        mailboxfull: [
+          %r/The user[(]s[)] account is temporarily over quota/,
+        ],
+        suspend: [
+          # http://www.naruhodo-au.kddi.com/qa3429203.html
+          # The recipient may be unpaid user...?
+          %r/The user[(]s[)] account is disabled[.]/,
+          %r/The user[(]s[)] account is temporarily limited[.]/,
+        ],
+        expired: [
+          # Your message was not delivered within 0 days and 1 hours.
+          # Remote host is not responding.
+          %r/Your message was not delivered within /,
+        ],
+        onhold: [
+          %r/Each of the following recipients was rejected by a remote mail server/,
+        ],
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'au EZweb: http://www.au.kddi.com/mobile/'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+      def headerlist;  return ['X-SPASIGN']; end
+      def pattern;     return Re0; end
+
+      # Parse bounce messages from au EZweb
+      # @param         [Hash] mhead       Message headers of a bounce email
+      # @options mhead [String] from      From header
+      # @options mhead [String] date      Date header
+      # @options mhead [String] subject   Subject header
+      # @options mhead [Array]  received  Received headers
+      # @options mhead [String] others    Other required headers
+      # @param         [String] mbody     Message body of a bounce email
+      # @return        [Hash, Nil]        Bounce data list and message/rfc822
+      #                                   part or nil if it failed to parse or
+      #                                   the arguments are missing
+      def scan(mhead, mbody)
+        return nil unless mhead
+        return nil unless mbody
+
+        match  = 0
+        match += 1 if mhead['from']    =~ Re0[:from]
+        match += 1 if mhead['subject'] =~ Re0[:subject]
+        match += 1 if mhead['received'].find { |a| a =~ Re0[:received] }
+        if mhead['message-id']
+          match += 1 if mhead['message-id'] =~ Re0[:'message-id']
+        end
+        return nil if match < 2
```

Comment:
__[rubocop]__
* Metrics/AbcSize: Assignment Branch Condition size for scan is too high. [143/30] [:ref](http://c2.com/cgi/wiki?AbcMetric)

<!-- Comment by [SideCI](https://sideci.com) -->